pai-management/bootstrap/frameworklauncher/frameworklauncher.yaml.template

```diff
@@ -65,9 +67,12 @@ spec:
       - name: launcher-log
         hostPath:
           path: {{ clusterinfo[ 'dataPath' ] }}/launcherlogs
-      - name: hadoop-config-volume
+      - name: hadoop-nm-config-volume
         configMap:
-          name: {{ clusterinfo[ 'hadoopinfo' ][ 'configmapname' ] }}
+          name: hadoop-node-manager-configuration
+      - name: hadoop-dn-config-volume
+        configMap:
+          name: hadoop-data-node-configuration
```

Comment:
hadoop-dn-configuration?

Reply:
Configmap name is hadoop-data-node-configuration. But the volume‘s name is hadoop-dn-config-volume.

Reply:
Then why in generate-config.sh the cp command is cp  /hadoop-nm-configuration/core-site.xml ? Where is the folder /hadoop-nm-configuration defined?

Reply:
3 Concept:
Configmap name： hadoop-node-manager-configuration
Volume Created by Configmap：hadoop-nm-config-volume
The path mounted by the volume：hadoop-nm-configuration